client/signup/steps/store-features/index.tsx

```diff
@@ -8,11 +8,13 @@ import paymentBlocksImage from 'calypso/assets/images/onboarding/payment-blocks.
 import wooCommerceImage from 'calypso/assets/images/onboarding/woo-commerce.svg';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { localizeUrl } from 'calypso/lib/i18n-utils/utils';
+import useBranchSteps from 'calypso/signup/hooks/use-branch-steps';
 import StepWrapper from 'calypso/signup/step-wrapper';
 import { saveSignupStep, submitSignupStep } from 'calypso/state/signup/progress/actions';
 import { getSite } from 'calypso/state/sites/selectors';
 import { shoppingBag, truck } from '../../icons';
 import SelectItems, { SelectItem } from '../../select-items';
+import { EXCLUDED_STEPS } from '../intent/index';
```

Comment:
Would it make more sense to pass the data from `EXCLUDED_STEPS.sell` as a prop/dependency from the `intent` step rather than importing the whole constant here? I don't think it makes a huge difference either way, but it sent me on a fun research journey re: importing/requesting vs. props so I thought I'd ask. :D

Reply:
That's a good question! I think I'll merge it as is and do some research on this topic as well 👍